proto/tendermint/budget/v1beta1/budget.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package cosmos.budget.v1beta1;
```

Comment:
Is it okay to leave the package as `cosmos.budget.v1beta1`? I'd suggest `crescent.budget.v1beta1` for consistency with other modules.

Reply:
The main purpose of this internalization is not to be consistency, but to lower the maintenance effort and risk of malfunction due to sdk version difference

when if change the package name, the [proto interface name](https://github.com/crescent-network/crescent/blob/5d72f6369b3bfbf753558ddfddda97974e571bbb/x/budget/types/budget.pb.go%23L177-L179) also changed, I think it is better to maintain the existing budget module status as much as possible even if the consistency is broken rather than verifying that this change causes state breaking

This PR should never include state breaking changes

Reply:
Got it, thanks